### ISSUE TITLE
Feature: Add validation script for validating tfvars/env files

### ISSUE THREAD

**Issue by mdrahman-suse** [rancher/distros-test-framework#152]
As a user I should be able to see any syntactical errors in tfvars or env files and log any errors if the docker image is stuck for any reason

Currently we lack any such logic which, if any syntactical error is present like missing quotes or comments, cause docker image creation to be stuck at one point without notifying why it failed. So we would like a add a script in the framework for perform that as an environment pre-check.

**Comment by VestigeJ** [comment]
Which of any or all of the environment variables are required?  
I'd like to label them as required and incorporate checking that those are set not nil before executing scripts.

**Comment by mdrahman-suse** [comment]
AFAIK These are required
```
# Framework config variables
ENV_PRODUCT={product}
ENV_TFVARS={product}.tfvars
ACCESS_KEY_LOCAL="pem_file.pem"
```
Should add the required ENV vars here: https://github.com/rancher/distros-test-framework/blob/main/docs/examples/.env.example
Need to re-visit these example files and update accordingly

**Comment by VestigeJ** [comment]
I was hoping missing a required variable was why I'm not able to run the latest pull from the repo (I used to run just via go directly without docker in the mix and this doesn't seem to work anymore).

**Comment by ShylajaDevadiga** [comment]
also PRODUCT/product

**Comment by mdrahman-suse** [comment]
> I was hoping missing a required variable was why I'm not able to run the latest pull from the repo (I used to run just via go directly without docker in the mix and this doesn't seem to work anymore). The recording from a few weeks ago is helpful and explains some of what I think I was dealing with.

Yes that is because the tfvars file had missing quotes or uncommented texts, which was causing docker to freeze on jenkins but locally worked fine
